app/src/androidTest/java/com/android/sample/createRecipe/RecipeAddImageScreenTest.kt

```diff
@@ -81,13 +80,14 @@ class RecipeAddImageScreenTest {
   }
 
   @Test
-  fun nextStepNotAvailable() {
+  fun clickNextWithoutPhotoShowsToast() {
     composeTestRule.setContent {
       RecipeAddImageScreen(
           navigationActions = navigationActions, createRecipeViewModel = createRecipeViewModel)
     }
 
-    composeTestRule.onNodeWithText("Next").assertIsNotEnabled()
+    // Simulate clicking the "Next" button without a photo
+    composeTestRule.onNodeWithText("Next").assertIsDisplayed().performClick()
```

Comment:
In the PR you mention checking if the toast appears. Did you mean by looking at it yourself during the test or is there supposed to be some logic checking that the toast has appeared?

Reply:
Im meant by looking at myself (honestly i didnt figure out how to test a context), does it really matter to add a test for this ? because i didnt find any other tests in the project using this